### PR TITLE
Switch to Qt 5.11.3 for all OS

### PR DESCRIPTION
**Pull request by DonLakeFlyer**
Also removed some references to things no longer available.

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
@hamishwillee I think I did this right with the version thing.

File: en/getting_started/README.md [hunk 1]
```diff
@@ -54,19 +54,20 @@ When installing, you must minimally select all Visual C++ components as shown:
...
+1. In the installer *Select Components* dialog choose: {{ book.qt_version }}.
    
    Then install just the following components: 
    - **Windows**: *MSVC 2015 32 bit*
    - **MacOS**: *macOS*
    - **Linux**: *Desktop gcc 64-bit*
    - All:
-     - *Qt Charts* and *Qt Remote Objects (TP)*
-     - *Android ARMv7* (to build Android)
+   - *Qt Charts* and *Qt Remote Objects (TP)*
```
Comment (hamishwillee) [inline en/getting_started/README.md]: This should have remained indented. I'll merge and fix,

Comment (DonLakeFlyer) [reply en/getting_started/README.md]: Oops, left some crap from trying to figure out why the list wasn't rendering properly.

Comment (hamishwillee) [reply en/getting_started/README.md]: OK. So right now this renders as below. If it needs to be anything else, let me know :-)

image

**Pull request merged**